### ISSUE TITLE
aria-hidden-focus: Ensures aria-hidden elements do not contain focusable elements 

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#3810]
## Version
4.12.1, npm

## Describe the bug

- When navigating transcript, attachment uploads only reads “1 attachment” when on ‘activity focus’
- After pressing enter on the attachment, nothing is read aloud by AT
- Aria-hidden value is set on the anchor inside the attachment

## Steps to reproduce

1. Open MockBot
2. type 'file', 'document-word', 'document-plain', and/or 'document-data-uri'
3. Chrome/NVDA: 
   - When navigating transcript, attachment uploads only reads “1 attachment” when on ‘activity focus’
   - Enter to interact is not read
   - After pressing enter on the attachment, nothing is read aloud by AT
   - Also repros https://github.com/microsoft/BotFramework-WebChat/issues/3804 behavior
   - Note: `aria-live` behavior works as expected
4. Edge/Narrator:
   - 'Enter to interact' is read as expected
   - Pressing enter on the attachment says "Nothing to interact" or something similar
   - `aria-live` behavior seems as expected
   

## Expected behavior
When focus is on the attachment, "1 attachment, Press enter to interact" should be read.
When `aria-selected` is on the attachment, content of the anchor should be read

## Additional context

[Bug]
